toolkit/scripts/telemetry_hopper/telemetry_hopper.py

```diff
@@ -0,0 +1,238 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import os
+import logging
+import grpc
+import signal
+import argparse
+from concurrent import futures
+from typing import Dict, Any, List, Optional
+
+from azure.monitor.opentelemetry.exporter import AzureMonitorTraceExporter
+from opentelemetry import trace
+from opentelemetry.trace.status import Status, StatusCode
+from opentelemetry.sdk.resources import Resource
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.trace import SpanContext, TraceFlags, TraceState
+from opentelemetry.proto.collector.trace.v1 import (
+    trace_service_pb2,
+    trace_service_pb2_grpc,
+)
+from opentelemetry.proto.trace.v1.trace_pb2 import Span as ProtoSpan
+from opentelemetry.proto.common.v1.common_pb2 import KeyValue
+
+SHUTDOWN_GRACE_PERIOD_SEC = 5
+
+# TODO: Replace with PME resource string when available
+AZURE_CONN_STR = "InstrumentationKey=c0b360fa-422d-40e5-b8a9-d642578f9fce;IngestionEndpoint=https://eastus-8.in.applicationinsights.azure.com/;ApplicationId=087d527c-b60e-4346-a679-f6abf367d0f0"
+
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger("image-customizer-telemetry")
```

Comment:
where do these logs go eventually? In other words, if one wants to see the logs, where can we find them?

Reply:
Thought about this more, I updated it to direct the log output to a dedicated log file so in the chance it needs to be inspected, the log output is stored. It still keeps it away from the stdout. I think this is better.